calico/reference/architecture/design/l3-interconnect-fabric.md

```diff
@@ -4,264 +4,130 @@ description: Understand considerations for implementing interconnect fabrics wit
-There are other options, and we will briefly touch on why we didn't
-include some of them in the [Other Options appendix](#other-options).
+In a {{site.prodname}} network, each compute server acts as a router for all of the endpoints that are hosted on that compute server. We call that function a vRouter. The data path is provided by the Linux kernel, the control
+plane by a BGP protocol server, and management plane by {{site.prodname}}'s on-server agent, *Felix*.
+
+Each endpoint can only communicate through its local vRouter, and the first and last *hop* in any {{site.prodname}} packet flow is an IP router hop through a vRouter. Each vRouter announces all of the endpoints it is attached to to all the other vRouters and other routers on the infrastructure fabric, using BGP, usually with BGP route reflectors to
+increase scale. A discussion of why we use BGP can be found in {% include open-new-window.html text='Why
+BGP?' url='https://www.projectcalico.org/why-bgp/' %}.
+
+Access control lists (ACLs) enforce security (and other) policy as directed by whatever cloud orchestrator is in use. There are other components in the {{site.prodname}} architecture, but they are irrelevant to the interconnect network fabric discussion.
+
+#### Overview of current common IP scale-out fabric architectures
+
+There are two approaches to building an IP fabric for a scale-out infrastructure. However, all of them, to date, have assumed that the edge router in the infrastructure is the top of rack (TOR) switch. In the {{site.prodname}} model, that function is pushed to the compute server itself.
+
+Furthermore, in most current virtualized environments, the actual endpoint is not addressed by the fabric. If it is a VM, it is usually encapsulated in an overlay, and if it is a container, it may be encapsulated in an overlay, or NATed by some form of proxy, such as is done in the {% include open-new-window.html text='Weave' url='http://www.weave.works/' %} project network model, or the router in standard {% include open-new-window.html text='docker' url='http://www.docker.io/' %} networking.
+
+The two approaches are: 
+
+**Routing infrastructure is based on some form of IGP** 
+
+Due to the limitations in scale of IGP networks, the {{site.prodname}} team does not believe that using an IGP to distribute endpoint reachability information will adequately scale in a {{site.prodname}} environment. However, it is possible to use a combination of IGP and BGP in the interconnect fabric, where an IGP communicates the path to the *next-hop* router (in {{site.prodname}}, this is often the destination compute server) and BGP is used to distribute the actual next-hop for a given endpoint. This is a valid model, and, in fact is the most common approach in a widely distributed IP network (say a carrier's backbone network). The design of these networks is somewhat complex though, and will not be addressed further in this technical note. ([note 3](#note-3)).
```

Comment:
Perhaps "See note 3"? Not a firm position, just following precedent for earlier notes

Same applies to all notes. (Solution may be to scrap the "see" for Note 2 above.